Latest/Model/AppLibrary.swift

```diff
@@ -47,20 +47,43 @@ class AppLibrary {
 	@objc func gatherAppBundles() {
 		self.appSearchQuery.disableUpdates()
 		
-		// Run metadata query to gather all apps
-		self.bundles = self.appSearchQuery.results.compactMap { item -> App.Bundle? in
-			guard let metadata = item as? NSMetadataItem,
-				let path = metadata.value(forAttribute: NSMetadataItemPathKey) as? String else { return nil }
-			
-			let url = URL(fileURLWithPath: path)
-			
-			// Only allow apps in the application folder and outside excluded subfolders
-			if !Self.applicationPaths.contains(where: { url.path.hasPrefix($0) }) || Self.excludedSubfolders.contains(where: { url.path.contains($0) }) {
-				return nil
+		let appSearchQueryResults = self.appSearchQuery.results
```

Comment:
I just tested this path by setting `appSearchQueryResults` to `[]`. I did not disable Spotlight on my Mac as I would have to disable SIP first. Do you have Spotlight disabled or do you know of a simpler way to test this?

I thought people reported endless activity indicator spinning with Spotlight disabled. So I thought `gatherAppBundles()` would never be called in this case, as the metadata query would never complete. I might be wrong about that though 😅

Reply:
This is actually your old code. I didn't intend to modify it, instead, it's now called as it did but only if Spotlight returns any results. So I don't think I should really touch it.